scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -100,6 +100,10 @@ def net_config(args):
     return get_config(values, os.path.join(TEMPLATE_DIR, "admin-net.xml"))
 
 
+def merge_dicts(d1, d2):
+    return dict(it.chain(d1.items(), d2.items()))
```

Comment:
it would be a terrible behavior change in the way I use it. I expect the arguments passed in to be unmutated. we can rename the function to make that more clearer if you want. 

I gathered this from http://treyhunner.com/2016/02/how-to-merge-dictionaries-in-python/
where this solution is the most pythonic / idiomatic recommendation.